### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by mugishab2020**

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    db_comment = crud.get_comment(db, comment_id=comment_id)
+    if db_comment is None:
+        raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND,
+                detail="Comment not found"
+                )
+    return crud.delete_comment(db=db, comment_id=comment_id)
+
+
+"""
```
Comment (the-1Riddle) [inline backend/app.py:183]: this needs to be a get request to the database not post, `post` should be on the signup process which is create account line:65

File: backend/app.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+@app.post("/login", response_model=schemas.Token)
```
Comment (the-1Riddle) [inline backend/app.py:184]: here

Comment (the1Riddle) [reply backend/app.py:184]: should be a post

**Review by the-1Riddle — COMMENTED** [review]
well

**Review by the1Riddle — APPROVED** [review]
the readme file looks cool

**Review by the-1Riddle — APPROVED** [review]
LGTM

**Pull request merged**